continuous-integration/starter-kits/compile-release-publish/java-maven/multimodule/multimodule-core/pom.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <name><!-- [INSERT NAME_MODULE_1 HERE] --></name>
+    <description><!-- [INSERT DESCRIPTION_MODULE_1 HERE] --></description>
+
+    <parent>
+        <groupId><!-- [INSERT GROUP_MM HERE] --></groupId>
+        <artifactId><!-- [INSERT ID_MM HERE] --></artifactId>
+        <version>${revision}</version>
+    </parent>
+
+    <groupId><!-- [INSERT GROUP_MM HERE] --></groupId>
+    <artifactId><!-- [INSERT MODULE_2 HERE] --></artifactId>
+    <packaging>jar</packaging>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <!-- VERSIONING -->
```

Comment:
This comment is a bit confusing in the context of other comments also using capitalized case, and requesting edits (i.e. INSERT XYZ HERE). Maybe regular comments are not capitalized? Or even remove redundant ones like this that don't really explain much?

Reply:
That's sort of a default requirement of the system, but I get your point. Perhaps Archetype structure will resolve some of this.